### PR TITLE
chore: Organized UI scripts into subdirectories for easier navigation.

### PR DESCRIPTION
**Pull request by paulhazen**
Organized the UI scripts into subdirectories so they can be more easily navigated, instead of having them all in one directory.

No changes to the code or functionality.

### CODE REVIEW

**Review by arthur740212 — APPROVED** [review]
Approved for the purpose of the PR.
Would like a follow up upm description modification, to not lose track of the path changes.

File: Assets/Scripts/UI/CustomInvites/UICustomInviteEntry.cs.meta [hunk 1]
```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 1873c9d1667766b4c9863e02341e3d79
+guid: ccb4e54585521f943b1423c25c1999dc
```
Comment (andrew-hirata-playeveryware) [inline Assets/Scripts/UI/CustomInvites/UICustomInviteEntry.cs.meta:2]: What caused the guid to change? Would it be possible to change it back?

File: Assets/Scripts/UI/CustomInvites/UICustomInvitesMenu.cs.meta [hunk 2]
```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: c58729c96d86a36438126f4fd7df9fd1
+guid: 1e0bb41d622c3b64995da5b28de50676
```
Comment (andrew-hirata-playeveryware) [inline Assets/Scripts/UI/CustomInvites/UICustomInvitesMenu.cs.meta:2]: This GUID as well should not have changed.

File: Assets/Scripts/UI/Leaderboard/UILeaderboardEntry.cs.meta [hunk 3]
```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: d1ba0aacd3cfe544a9811cab49c42362
+guid: 624cf320b5d6584448da00427b484102
```
Comment (andrew-hirata-playeveryware) [inline Assets/Scripts/UI/Leaderboard/UILeaderboardEntry.cs.meta:2]: Same GUID change here

File: Assets/Scripts/UI/Leaderboard/UILeaderboardMenu.cs.meta [hunk 4]
```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: f90663805b3a98246a05b5a22bc3c1da
+guid: 80add8f071caba44f9e3bbe8d2d16c92
```
Comment (andrew-hirata-playeveryware) [inline Assets/Scripts/UI/Leaderboard/UILeaderboardMenu.cs.meta:2]: Same GUID issue here.

File: Assets/Scripts/UI/NewFolder2.meta
Comment (andrew-hirata-playeveryware) [inline Assets/Scripts/UI/NewFolder2.meta:1]: I think this file was added in error.

File: Assets/Scripts/UI/Peer2Peer/UIPeer2PeerMenu.cs.meta [hunk 5]
```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 5c55cb3e7656c234595e8d4e18d76192
+guid: 38c49c5dd3cc3d64882d90a3280196fa
```
Comment (andrew-hirata-playeveryware) [inline Assets/Scripts/UI/Peer2Peer/UIPeer2PeerMenu.cs.meta:2]: This GUID also seems to have changed.

File: Assets/Scripts/UI/Peer2Peer/UIPeer2PeerParticleController.cs.meta [hunk 6]
```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 133c983cf907a2e439144fbc92f76e6c
+guid: 24431efb94b611c4a8699ed0f6bd1dca
```
Comment (andrew-hirata-playeveryware) [inline Assets/Scripts/UI/Peer2Peer/UIPeer2PeerParticleController.cs.meta:2]: This file also has a changed GUID.

File: Assets/Scripts/UI/Peer2Peer/UIPeer2PeerParticleLifetimer.cs.meta [hunk 7]
```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 27355e56a1850c349b621cd9008ae246
+guid: 1384ebf9c8db65e409b84c1b803e9f86
```
Comment (andrew-hirata-playeveryware) [inline Assets/Scripts/UI/Peer2Peer/UIPeer2PeerParticleLifetimer.cs.meta:2]: GUID change.

**Review by andrew-hirata-playeveryware — CHANGES_REQUESTED** [review]

**Comment by paulhazen** [comment]
Closing PR in order to make file changes / moves within the Unity editor so that GUIDs don't change.

**Pull request closed without merging**